.github/workflows/deploy.yml

```diff
@@ -26,14 +26,14 @@ jobs:
     strategy:
       matrix:
         # we could do it also as matrix but it takes longer
-        #python-version: ["3.11","3.10","3.9","3.8","3.7"]
+        #python-version: ["3.11","3.10","3.9","3.8"]
         #--- Build and deploy Linux wheels using manylinux containers ---
         # - build manylinux2010 x64 and i686
         # - build manylinux2014 x64
         # - build manylinux_2_28 x64
         # avoiding manylinux2014_i686 because does not provide librsync-devel
         # manylinux_2_24 is already deprecated
-        many-linux: [2010_x86_64, 2010_i686, 2014_x86_64, _2_28_x86_64]
+        many-linux: [2014_x86_64, _2_28_x86_64]
```

Comment:
I'm not fluent with ManyLinux. Correct me if I'm wrong. I tought the new convention is to use _2_28_x86_64 where _2_28 is compatible with glibc >=2.28. Is it required to keep 2014_x86_64 ?

Reply:
My understanding is, yes, for slightly older platforms: https://github.com/pypa/manylinux (e.g. CentOS 8 with Python3.8).